test/unit/aggregate_store_test.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 require_relative '../test_helper'
-require 'test_after_commit'
+Aggregate::ActiveRecordHelpers::Version.if_version(
+  active_record_4: -> { require 'test_after_commit' }
```

Comment:
Is this explicitly available in Rails 5+ then?

Reply:
I believe so. When I run it with appraisal rails-5 I get `after_commit testing is baked into rails 5, you no longer need test_after_commit gem (RuntimeError)`. So I kept it only for 4